src/components/Filters/FilterRow/FilterRow.tsx

```diff
@@ -58,6 +58,15 @@ const SplitField = styled.div<SplitFieldProps>`
   }
 `;
 
+const Units = styled.span`
```

Comment:
why not `Text` component with `md` size?

Reply:
Sure, I forget for `Text`